src/main/kotlin/racingcar/misc/Util.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar.misc
+
+import java.util.Random
+
+class Util {
+    companion object {
```

Comment:
이런경우, object 로 구현하시면 좋을거같아요!
```suggestion
object Util {
```

Reply:
활용할수 있는 기능들을 Util 클래스로 묶으셨군요! 
재활용할수 있다는 장점이 있을수 있으나,
Util을 사용하는 클래스와의 강한 의존성이 생기기도 하고 ( Wolrd 객체 )
Util이라는 모호한 범위의 객체라 관리하기도 힘들어지는것 같단 생각이 들어요 
다른 방법을 고민해보셔도 좋을거같아요 😄

Reply:
Util 클래스를 삭제하고, tryAndRerun 함수는 HandleException.kt 파일에 두었고, Random은 RandomNumberGenerator 클래스에 필요한 로직이라 판단하여 이동하였습니다.